docs/spec/spec.md

```diff
@@ -38,17 +38,7 @@ The command initializes the bal project with the persistence layer. This command
    This directory should contain all data model definition files. This file will define the required entities as per the [`persist` specification](https://github.com/ballerina-platform/module-ballerina-persist/blob/main/docs/spec/spec.md#2-data-model-definition)
 2. Create a model definition file in persist directory
    It will create a file with same name as the package name if no files are present in the `persist` directory. It will create an empty file with required imports(`import ballerina/persist as _;`).
-3. Create database_configurations.bal file inside the `default` module in the generated directory if the definition filename inside the `persist` directory is the same as the Ballerina package name. If not, create the bal file inside the submodule named as the definition file name.
-   This file contains configuration variables to initialize the data stores
-    ```ballerina
-    import ballerinax/mysql.driver as _;
-
-    configurable int port = ?;
-    configurable string host = ?;
-    configurable string user = ?;
-    configurable string database = ?;
-    configurable string password = ?;
-   ```
+3. Create the generated directory. If the definition filename inside the `persist` directory is different from the Ballerina package name, create a submodule named as the definition file name.
```

Comment:
generated directory and submodule should also create at generate command and the module name should be read from the Ballerina.toml file. In the init command, we only populate Ballerina.toml file with the provider and the module name.

Reply:
updated in [c5567e7](https://github.com/ballerina-platform/persist-tools/pull/148/commits/c5567e7273ef9c55ed1e47da998f94e65cf3a506)